src/sass/components/_about.scss

```diff
@@ -0,0 +1,93 @@
+.about {
+  background-color: $pageBg;
+  background: url('../images/sectionbg1_1x.png') no-repeat;
+  background-size: contain;
+  background-position: center bottom;
+
+  @media (min-width: 768px) {
+    background-image: none;
+  }
+  @media (min-width: 1200px) {
+    background-image: url('../images/sectionbg1_3x.png');
+    background-position-y: 140%;
+  }
+
+  &__tittle-2th {
+    @include font-mixin(Titan One, null, 34, 39px, null, $brand-red);
```

Comment:
font-family musisz dać w pojedynczy cudzysłów: 'test'
font-weight ma 400, a nie null
font-size popraw na 34px
line height obliczasz dzieląc line height/font-size, czyli 39/34, to wychodzi ~1.15 i tak zapisz
letter- spacing ma 0.04em, popraw